### DISCUSSION TITLE
Energy dashboard entity new/edit filter not working

### DISCUSSION THREAD

**Discussion by chris2172** [home-assistant/frontend#17290] [Dashboard]
When editing an item in the energy dashboard the entity is no longer shown. when you try to re enter the entity id sensor.gas_kwh for example the filter will show that entity then reset to all entities within 1 second. this makes it almost imposable to create an energy monitoring device. Not sure if this is a specific web browser or HA problem, I have the same issue using Chrome & Edge browsers.

**Comment by karwosts** [comment]
As this is a bug report, can you please fill out the full issue template and file this in the issues section if you are still having problems? 

This behavior was changed/fixed in 2023.7, and I do not know if you have the correct version or not. More information/screenshots/screen recordings would be helpful.
